src/utils.ts

```diff
@@ -1,22 +1,15 @@
 import {Dimensions} from 'react-native';
 import * as animatable from 'react-native-animatable';
+import {CustomAnimation} from 'react-native-animatable';
+import {Animations} from './types';
+import {Animation} from 'react-native-animatable';
 
 const {height, width} = Dimensions.get('window');
 
 export const initializeAnimations = () => {
-  const makeSlideTranslation = (translationType, fromValue, toValue) => {
-    return {
-      from: {
-        [translationType]: fromValue,
-      },
-      to: {
-        [translationType]: toValue,
-      },
-    };
-  };
   // Since react-native-animatable applies by default a margin of 100 to its
   // sliding animation, we reset them here overriding the margin to 0.
-  const animationDefinitions = {
+  const animationDefinitions: Record<string, CustomAnimation> = {
```

Comment:
You always learn something new. 
`Record` is new to me, I've always used `{[string]: CustomAnimation}`. Awesome.